redash/handlers/api.py

```diff
@@ -69,6 +70,11 @@ def json_representation(data, code, headers=None):
 api.add_org_resource(QueryRefreshResource, '/api/queries/<query_id>/refresh', endpoint='query_refresh')
 api.add_org_resource(QueryResource, '/api/queries/<query_id>', endpoint='query')
 
+api.add_org_resource(AccessListResource, '/api/access/<object_type>/<object_id>', endpoint='list_access')
+api.add_org_resource(AccessGrantResource, '/api/access/<object_type>/<object_id>', endpoint='grant_access')
+api.add_org_resource(AccessRevokeResource, '/api/access/<object_type>/<object_id>', endpoint='revoke_access')
```

Comment:
How about we name this resource as `access_permission`?

Also you can merge the last 3 classes into one (assuming you change the access check to GET). So it becomes `AccessPermissionListResource` and `AccessPermissionResource`.